server/migrations/20190318001110-channelsTable.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    await queryInterface.createTable('Channels', {
+      id: {
+        type: Sequelize.UUID,
+        primaryKey: true,
+        defaultValue: Sequelize.fn('uuid_generate_v4'),
```

Comment:
Why this instead of the previous way (which I think was done in the model layer)?

Reply:
isn't it better to do the generation on the database side? you can use it even in raw queries
then we can migrate other tables to this default value too

Reply:
it's like single way of truth

Reply:
I like that, makes sense, just checking.